test/integration_test/specs/kubevirt-fedora/fedora-datavolume-template.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: fedora-template-pvc
```

Comment:
Do we need "-pvc" in the name? It might be confusing since this is not a pvc object. We could use "fedora-golden-image" or something like that.